docs/blog/e18e.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: e18e (Ecosystem Performance) - A new community initiative
+title: e18e (Ecosystem Performance), a new community initiative
```

Comment:
I don't think we need to change this

Reply:
A capital letter after an en-dash is not correct. I realize this may seem like nit picking, but this is the job of a professional copy editor. I'm just trying to make your texts look more professional, donating about an hour of my time to do so, because I believe in your mission.